README.md

```diff
@@ -10,11 +15,11 @@ There is ongoing work for refactoring cloud providers out of the upstream reposi
 
 ## Quickstart
 
-Get started with Cloud controller manager for vSphere with Kubeadm with this [quickstart](https://cloud-provider-vsphere.sigs.k8s.io/tutorials/kubernetes-on-vsphere-with-kubeadm.html).
```

Comment:
Did https://cloud-provider-vsphere.sigs.k8s.io/tutorials/kubernetes-on-vsphere-with-kubeadm.html not work?

Reply:
the link is working, but I don't think it directs to the right place
I think we would want to visit the page that teaches deploying Cloud controller manager for vSphere with Kubeadm, right? 🤔

Reply:
I think https://github.com/kubernetes/cloud-provider-vsphere/blob/master/docs/book/tutorials/kubernetes-on-vsphere-with-kubeadm.md is the doc we want to share as a tutorial with kubeadm, and https://cloud-provider-vsphere.sigs.k8s.io/tutorials/kubernetes-on-vsphere-with-kubeadm.html is the docs site version that.